### ISSUE TITLE
Temporary pan/zoom on Labels layer lasts <1-2 seconds

### ISSUE THREAD

**Issue by hectormz** [napari/napari#1143]
## 🐛 Bug

When in brush/fill/pick mode on a Labels layer, holding space bar temporarily enables pan/zoom mode, but reverts to the original mode after a second or two

## To Reproduce

1. Create Labels layer
2. Select pick/fill/brush mode
3. Hold space bar
    - Mode switches to pan/zoom for < 1-2 sec
    - Mode switches back to original mode despite space bar still held
    - No pan/zooming may have happened before reverting back to original mode

## Environment

```
napari: 0.2.12+51.g6ea95a9
Platform: Windows-10-10.0.19041-SP0
Python: 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)]
Qt: 5.14.1
PySide2: 5.14.1
NumPy: 1.18.1
SciPy: 1.4.1
Dask: 2.12.0
VisPy: 0.6.4

GL version: 4.6.0 NVIDIA 441.45
MAX_TEXTURE_SIZE: 32768

Plugins:
- mat-images: 0.1.0
```

**Comment by jni** [comment]
This happens to me on Linux also. I guess holding space only works on Mac. Possibly something to do with how key-hold events are sent or detected by Qt on different platforms... See #133 :joy:

**Comment by hectormz** [comment]
@jni Ah, I didn't see that before. From doing some basic testing, it looks like the holding the space bar is only registering as a single event, while holding any of the arrow keys results in repeated events. I'm not sure if @kne42 has any insight into this, although it seemingly works fine on OS X?

**Comment by sofroniewn** [comment]
This has been closed by #1350
